src/core/turn-order.test.ts

```diff
@@ -451,6 +451,53 @@ describe('setActivePlayers', () => {
     expect(state.ctx.activePlayers).toBeNull();
   });
 
+  test('setActivePlayers and setStage', () => {
```

Comment:
Let’s use a slightly more descriptive test name:

```suggestion
  test('set stages to Stage.NULL', () => {
```